### PR TITLE
ROC-3375 and ROC-3336 Defendant handoff guidance pages

### PR DESCRIPTION
**Pull request by robertparkinson**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-3375
https://tools.hmcts.net/jira/browse/ROC-3336

### Change description

Update hand off pages with new content

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/features/response/views/eligibility/send-your-response-by-email.njk [hunk 1]
```diff
@@ -16,9 +16,84 @@
         </div>
       </div>
 
-
       <h2 class="heading-medium">{{ t('Follow these steps') }}</h2>
 
+      <ul class="list list-number">
+        <li>
+          {% if draft.response.type === ResponseType.FULL_ADMISSION %}
+            {{ t('Download the <span class="bold-small">admission form.</span>') | safe }}
```
Comment (timja-kainos) [inline src/main/features/response/views/eligibility/send-your-response-by-email.njk]: are these supposed to be linked?

Comment (robertparkinson) [reply src/main/features/response/views/eligibility/send-your-response-by-email.njk]: links are now at the bottom of the page

Comment (robertparkinson) [reply src/main/features/response/views/eligibility/send-your-response-by-email.njk]: for a visual: 
https://hmcts-cmc-prototype.herokuapp.com/defendant/full/d-5/task-list/do-you-owe-money/forms?handoffs=4

**Comment by rlewan** [comment]
Martin reviewed this content change and is happy with it, merging.

**Pull request merged**